src/components/date/date.md

```diff
@@ -0,0 +1,42 @@
+For design and accessibility functionality, see [Form Controls > Date documentation](https://www.notion.so/Date-eb1b5fc8ce324cc39eaaa28772d055da).
```

Comment:
We're going to try to flesh out the Notion document this week. We'll see if it's better -- at the very least we can have design and QA write up their expectations around the component.